content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-13.md

```diff
@@ -33,15 +33,15 @@ This page explains how to upgrade a Kubernetes cluster created with `kubeadm` fr
 
 {{% capture steps %}}
 
-## Upgrade the control plane
+## Upgrade the control plane node
 
-1.  On your master node, upgrade kubeadm:
+1.  On your control plane node, upgrade kubeadm:
 
```

Comment:
Determine what version of kubeadm you have installed.
```
dpkg-query --show kubeadm
kubeadm	1.12.0-00
```
```
rpm -qa kubeadm
kubeadm-1.12.0-0.x86_64
```
Then determine the most recent next minor version of the kubeadm.
```

```

```
yum list --showduplicates  kubeadm --disableexcludes=kubernetes
[truncated]
kubeadm.x86_64                  1.12.0-0                   kubernetes 
kubeadm.x86_64                  1.12.1-0                   kubernetes 
kubeadm.x86_64                  1.12.2-0                   kubernetes 
kubeadm.x86_64                  1.12.3-0                   kubernetes 
kubeadm.x86_64                  1.13.0-0                   kubernetes

Reply:
> apt-cache policy kubeadm
> ...
> yum list --showduplicates  kubeadm --disableexcludes=kubernetes

do we need to update first e.g. `apt update` (not sure what the yum altermative is for update)

> In this case I could upgrade safely to 1.12.3 or to 1.13.0

what if the next candidate is 1.14.0?

for apt something like this can be done:
> apt-get install -y kubelet=1.13.0-0 kubeadm=1.13.0-0

we use this trick for k-a:
https://github.com/kubernetes/kubernetes-anywhere/blob/kubeadm-e2e/phase2/kubeadm/configure-vm-kubeadm.sh#L57-L58
to get the latest stable.

not sure how to do it for yum.